web/src/gear_menu.ts

```diff
@@ -174,6 +175,13 @@ export function initialize(): void {
                 e.stopPropagation();
             });
 
+            $popper.on("click", ".convert-demo-organization", (e) => {
+                popover_menus.hide_current_popover_if_visible(instance);
+                e.preventDefault();
+                e.stopPropagation();
+                demo_organizations_ui.do_convert_demo_organization();
```

Comment:
`do_convert_demo_organization` appears to be poorly named -- `do_` in our codebase typically means it's after any checks/dialogs, and that function presents a modal to fill in the subdomain etc.